testsuite/integration-arquillian/util/src/main/java/org/keycloak/testsuite/utils/io/IOUtil.java

```diff
@@ -184,7 +184,10 @@ public static void modifyDocElementValue(Document doc, String tagName, String re
             return;
         }
 
-        node.setTextContent(node.getTextContent().replaceFirst(regex, replacement));
+        if (node.getTextContent().contains(regex)) {
```

Comment:
This changes semantics. `contains` uses direct comparison, `replaceFirst` uses regexps. Please stick to the regexps, they are richer in expressiveness.

Feel free to use `".*infinispan\\.InfinispanSessionCacheIdMapperUpdater"` instead of `"wildfly.infinispan.InfinispanSessionCacheIdMapperUpdater"` on L64

Reply:
@hmlnarik thanks, incorporated